server/src/main/assets/css/partials/_main.scss

```diff
@@ -14,6 +14,10 @@ main {
     flex: 1;
 }
 
+.copyable-incantation {
+  user-select: all;
```

Comment:
This has the nice effect of the user selecting the entire text sample, rather just a word, when they click on it - at that point it's very easy to copy & paste the entire expression, so it's a nice complement (or possibly even replacement) to the Javascript 'Copy' button.

Reply:
Nice trick!